src/main/pages/che-7/end-user-guide/proc_defining-maven-settings-xml-file-across-workspaces.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// using-maven-artifact-repositories
+
+[id="proc_defining-maven-settings-xml-file-across-workspaces_{context}"]
+= Defining Maven `settings.xml` file across workspaces
+
+To specify your own `settings.xml` file across workspaces, use the Secret with `settings.xml` and define labels and annotation to mount it as a file (see link:{site-baseurl}che-7/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/#mounting-a-secret-as-a-file-into-a-workspace-container_mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container[Mounting a secret as a file into a workspace container]).
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: maven-settings-secret
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: workspace-secret
+  annotations:
+    che.eclipse.org/target-container: maven
+    che.eclipse.org/mount-path: /home/user/.m2
+    che.eclipse.org/mount-as: file
+type: Opaque
+stringData:
+  settings.xml: |-
+    <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Comment:
should it be base64 encoded?

Reply:
Yes, that's another option. I would rather put here the link to k8s documentation to avoid sidetracking.

Reply:
mm. I think that might be problematic to figure out all steps for users in case if he wants to add credentials that contains some special characters.  Can we get some concrete examples with concrete steps?

Reply:
yes, included in example